app/models/member.rb

```diff
@@ -0,0 +1,5 @@
+class Member < ApplicationRecord
+
+  has_many :user, inverse_of: :member
+
```

Comment:
Do you have the admin screens for creating/managing members and users on your list of todos?  Let me know if I need to make a card or if there's another plan.

Reply:
we have https://trello.com/c/SlRiarkx/1248-antech13-create-a-power-user-role I added a line saying that power user can also edit/delete other user's accounts.